types/ember-cli-fastboot/services/fastboot.d.ts

```diff
@@ -0,0 +1,83 @@
+import Service from "@ember/service";
+
+export interface FastbootRequest {
+  /** The cookies for the current request. */
+  cookies: Record<string, unknown>;
+
+  /** The headers for the current request. */
+  headers: Pick<Headers, "has" | "get">;
+
+  /**
+   * The host of the request that the current FastBoot server is responding to
+   * (`example.com` or `localhost:3000`).
+   * Retrieving host will error on 2 conditions:
+   * 1. you do not have a hostWhitelist defined
+   * 2. the Host header does not match an entry in your hostWhitelist
+   */
+  host: string;
+
+  /** The query parameters for the current request, as a key-value pair. */
+  queryParams: Record<string, unknown>;
+
+  /**
+   * The path (`/` or `/some-path`) of the request that the current FastBoot
+   * server is responding to.
+   */
+  path: string;
+
+  /**
+   * The protocol (`http:` or `https:`) of the request that the current
+   * FastBoot server is responding to.
+   */
+  protocol: string;
+}
+
+export interface Shoebox {
+  put(key: string, value: unknown): void;
+  retrieve(key: string): unknown;
+}
+
+export class FastBoot extends Service {
```

Comment:
Whoops, my bad, missed this before:

```suggestion
export default class FastBoot extends Service {
```